auth/integration_test/Podfile

```diff
@@ -7,6 +7,11 @@ target 'integration_test' do
   pod 'Firebase/Auth', '7.11.0'
```

Comment:
Instead of leaving "platform:ios" at global scope, let us bring it into the ios target.

Reply:
Was a quick change to make.